src/librustc_query_system/query/plumbing.rs

```diff
@@ -645,6 +647,7 @@ where
 /// side-effects -- e.g., in order to report errors for erroneous programs.
 ///
 /// Note: The optimization is only available during incr. comp.
+#[inline(never)]
```

Comment:
why the `#[inline(never)]` marker here? Did you observe a need for it on some benchmark locally?

Reply:
(no, your description says your rig is not suitable for local benchmarking, so that cannot be the answer...)

Reply:
I don't remember. I can remove it.